shared/vehicles.lua

```diff
@@ -5349,7 +5349,7 @@ QBShared.Vehicles = {
         ['shop'] = 'truck',
     },
     ['mixer2'] = {
-        ['name'] = 'Mixer2',
+        ['name'] = 'HVY Cement Mixer 2',
```

Comment:
```suggestion
        ['name'] = 'Cement Mixer 2',
```
The brand is not needed in the actual label of the vehicle.